Assets/Tests/InputSystem.Editor/InputActionsEditorTests.cs

```diff
@@ -173,5 +189,55 @@ public IEnumerator CanDeleteActionMap()
         Assert.That(m_Window.currentAssetInEditor.actionMaps[0].name, Is.EqualTo("First Name"));
         Assert.That(m_Window.currentAssetInEditor.actionMaps[1].name, Is.EqualTo("Third Name"));
     }
+
+    [UnityTest]
+    public IEnumerator CanRenameAction()
```

Comment:
great to see a test for this! There were a lot of cases on renaming and focus issues already.